### DISCUSSION TITLE
[BP#66] Suppression de la pratique « Utiliser la simple quote (') au lieu du guillemet (") »

### DISCUSSION THREAD

**Discussion by rlemaire** [cnumr/best-practices#17] [Bonnes pratiques] [answered]
La différence n'est pas très grande en terme d'instructions économisées.

**Comment by ACTLEM** [comment]
Voir https://github.com/cnumr/best-practices/discussions/98. Merci à @fzaninotto pour les résultats des tests.

**Comment by ACTLEM** [comment, marked as answer]
https://github.com/cnumr/best-practices/issues/129
